Dockerfile

```diff
@@ -1,17 +1,11 @@
 FROM dmscid/lewis-depends:latest
 
-# Copying this separately from the rest of lewis allows the
-# pip install step to be cached until the requirements change.
-# Rebuilding is faster and doesn't require an internet connection.
-COPY requirements.txt /lewis/
+# Lewis source is pulled in from current directory (note .dockerignore filters out some files)
+COPY . /lewis
 
 # Install any missing PIP requirements, clear PIP cache, and delete all compiled Python objects
-RUN pip install -r lewis/requirements.txt && \
+RUN pip install -e /lewis && \
```

Comment:
Unfortunately, moving `COPY . /lewis` up above this in order to make `pip install -e /lewis` work also means breaking the...
>Rebuilding is faster and doesn't require an internet connection.

... "feature" mentioned in the comment above.

What happened before was that docker cached these two steps:
```dockerfile
COPY requirements.txt /lewis/
RUN pip install -r lewis/requirements.txt && \
    ...
```
... when you ran `docker build` for the first time. After that, if you ran `docker build` again, it just used the cached results and didn't re-execute them unless `requirements.txt` changed. Since `COPY . /lewis` came after, a change to the source wouldn't invalidate that cache. Now, any change to the source also means all the requirements need to be downloaded and installed again.

But I think that's not too tragic / a price worth paying for the benefits of using `pip install -e`?